src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6169,6 +6309,21 @@ export class StudyViewPageStore
                         profileType: newChart.profileType,
                     } as any);
                     this.chartsDimension.set(uniqueKey, { w: 2, h: 1 });
+                } else if (
+                    newChart.mutationOptionType &&
+                    newChart.mutationOptionType ===
+                        MutationOptionConstants.MUTATION_TYPE
```

Comment:
correct me if i'm wrong but it seems like we use mutation_type, mutation_event and mutationData terminology interchangeably.  is it possibly to make this consistent?

Reply:
We make mutation_type as the final say both in frontend and backend.

Reply:
let me change all occurences in frontend